apex_dual_test.go

```diff
@@ -14,19 +14,19 @@ import (
 
 func setupEmptyLookupFuncs(gw *Gateway) {
 	if resource := gw.lookupResource("HTTPRoute"); resource != nil {
-		resource.lookup = func(_ []string) []netip.Addr { return []netip.Addr{} }
+		resource.lookup = func(_ []string) (results []netip.Addr, txts []string) { return []netip.Addr{}, []string{} }
```

Comment:
txts, should and could be anything else but `txt` it should be whatever is not an IP address